src/providers/vertexUtil.ts

```diff
@@ -92,10 +93,31 @@ export interface Palm2ApiResponse {
   ];
 }
 
-export function maybeCoerceToGeminiFormat(contents: any) {
+const GeminiFormatSchema = z.object({
```

Comment:
Isn't `contents` potentially a list and `parts` too?
https://ai.google.dev/gemini-api/docs/quickstart?lang=rest